### PR TITLE
Feature/4959 new startup script

### PR DESCRIPTION
**Pull request by stuart-bass-bjss**

### CODE REVIEW

**Review by spartanbear — APPROVED** [review]
(Y)

File: local-kubernetes/README.md [hunk 1]
```diff
@@ -2,17 +2,49 @@
...
-To run the system, you need to create a secret to access the private container registry.
+## Set Up Kubernetes
+
+To begin, make sure you have kubernetes running locally as per the [Kubernetes Dev Setup Instructions](../Docs/DevSetup/local-k8s-setup.md)
+
+Create the buying catalogue namespace - `kubectl apply -f local-namespace.yml`
+
+To run the system, you need to create a secret in kubernetes to access the private container registry, as per the [Connect Local Kubernetes with our Private Container Registry Instructions](../Docs/DevSetup/local/k8s-private-registry.md).
+
+You will also need a secret for the local instances of sql server - `kubectl create secret generic mssql --from-literal=SA_PASSWORD="<YOUR PASSWORD>" -n buyingcatalogue`
```
Comment (PhysicsJon) [inline local-kubernetes/README.md]: Needs a note to explain minimum password length (8 characters according to SQL Server)

Comment (stuart-bass-cgi) [reply local-kubernetes/README.md]: added

**Review by PhysicsJon — APPROVED** [review]

File: local-kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- [Marketing Pages](http://localhost:3002/supplier/solution/100000-001/preview) on port 3002
+- [Public Browse](http://localhost:3000/) on port 3000
+
+Run `tear-down-environment{.sh|ps1}` to tear down the environment.
+
+## Running the dashboard
+
+You may also wish to use the dashboard to view what is happening in Kubernetes (instructions below from [collabnix.com](https://collabnix.com/kubernetes-dashboard-on-docker-desktop-for-windows-2-0-0-3-in-2-minutes/))
+- Apply the dashboard yaml - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v1.10.1/src/deploy/recommended/kubernetes-dashboard.yaml`
+- Run the proxy - `kubectl proxy`
```
Comment (talawson05) [inline local-kubernetes/README.md]: can you move this task to the bottom of the sequence, so that you do the token first?

Comment (stuart-bass-cgi) [reply local-kubernetes/README.md]: changed

File: local-kubernetes/README.md [hunk 3]
```diff
@@ -43,6 +42,7 @@ kubectl config set-credentials docker-desktop --token="${TOKEN}"
 TOKEN=`kubectl -n kube-system describe secret default | grep token: | awk '{print $2}'`
 kubectl config set-credentials docker-desktop --token=$TOKEN
 ```
+- Run the proxy - `kubectl proxy`
 - Browse to [http://:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/](http://:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/)
```
Comment (PhysicsJon) [inline local-kubernetes/README.md]: This link give a `about:blank#blocked` response when clicked.

**Review by PhysicsJon — COMMENTED** [review]
Link failing

**Review by PhysicsJon — APPROVED** [review]

File: local-kubernetes/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  - [DAPI](http://localhost:5101/swagger) on port 5101 
+  - Azurite on ports 10000 (blob),10001 (queue),10002 (table), with development user default (access with [Azure Storage Explorer](https://azure.microsoft.com/en-gb/features/storage-explorer/))
+- Buying Catalogue Service
+  - [BAPI](http://localhost:5100/swagger) on port
+  - SQL Server on port 1450 (log in localhost,1450 in [SSMS](https://docs.microsoft.com/en-us/sql/ssms/download-sql-server-management-studio-ssms?view=sql-server-ver15))
+- [Marketing Pages](http://localhost:3002/supplier/solution/100000-001/preview) on port 3002
+- [Public Browse](http://localhost:3000/) on port 3000
+
+Run `tear-down-environment{.sh|ps1}` to tear down the environment.
+
```
Comment (talawson05) [inline local-kubernetes/README.md:29]: is it worth having a section for how to connect Azure storage explorer to the local azurite container, and an SQL client to the local DB?

**Review by talawson05 — APPROVED** [review]

**Pull request merged**